projects/make-recall-decisions-and-delius/deploy/database/access.yml

```diff
@@ -5,5 +5,5 @@ database:
 
```

Comment:
Think you'll need to add the table access here to enable writing to the contact / contact_alert tables